app/Models/WaitingRoom.php

```diff
@@ -15,4 +15,62 @@ public function users()
     {
         return $this->belongsToMany(User::class);
     }
+
+    /*
+     * Equally splits the users of a waiting room into arrays
+     * representing competitions.
+     */
+    public function getDefaultSplit()
+    {
+        $users = $this->users;
+
+        // Get the size of the waiting room
+        $roomSize = count($users);
+
+        // Determine the amount of competitions to make
```

Comment:
Not important for commit, but don't think we need that many comments on all these steps ;) Just leave comments on the ones that might seem a little cryptic. :wink: